packages/frontend/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1084,6 +1090,26 @@ watch(isModelValueExpression, async (isExpression, wasExpression) => {
 	}
 });
 
+const unwatchParameterOptions = watch(
```

Comment:
this is a temporary change to investigate the reason for this? Can we add a comment explaining this and referencing the sentry issue?